### ISSUE TITLE
[Feature]: Removal of resolution hint on changing window size from Media Player Window

### ISSUE THREAD

**Issue by dagandrew** [sircharlo/meeting-media-manager#3723]
### Is your feature request related to a problem?

When sharing on Zoom and going fullscreen or adjusting Media Player Window size you can see resolution hint and it's distracting. 

### Describe the solution you'd like

Remove the hint all together

### Describe alternatives you've considered

Put it somewhere in the Manager, for example here 
![image](https://github.com/user-attachments/assets/df7470ae-70cb-4ad8-bc90-ba0360864b19)

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing issues to make sure my feature request is not a duplicate.
- [X] I confirm that I have searched through the existing discussions to make sure my feature request is not a duplicate.

**Comment by e-Feitosa** [comment]
> When sharing on Zoom and going fullscreen or adjusting Media Player Window size you can see resolution hint and it's distracting.

A question here reading your post, don't you use OBS Studio?

**Comment by dagandrew** [comment]
> > When sharing on Zoom and going fullscreen or adjusting Media Player Window size you can see resolution hint and it's distracting.
> 
> A question here reading your post, don't you use OBS Studio?

No, I don't use it. Just straight there, we have only-zoom meetings

**Comment by sircharlo** [comment]
I suppose the size notification is not incredibly useful. Removed from the media window in 3926c86b1aec73dac60a38b41f9994043df02180 and added as a tooltip on the "Window" button in the Displays popup in the main window instead.
